docs/notebooks/FastText_Tutorial.ipynb

```diff
@@ -300,8 +300,8 @@
       ]
      },
      "execution_count": 7,
-     "output_type": "execute_result",
-     "metadata": {}
+     "metadata": {},
```

Comment:
Sorry.
I had opened it up in jupyter which made the changes to it automatically.
removed it.